packages/blade/src/tokens/theme/bankingTheme.ts

```diff
@@ -2141,75 +2144,14 @@ const colors: ColorsWithModes = {
   },
 };
 
-const shadows: Shadows = {
```

Comment:
Hmm 🤔 won't this be a breaking change? Sure internally we are migrating it, but what if consumer is already using these tokens? theme.shadows.offsetX etc?

Reply:
no one is using them since it was not even in usable state :D

Reply:
:p okay